src/holon/models/rule_actions/rule_action_change_attribute.py

```diff
@@ -23,8 +23,9 @@ class RuleActionChangeAttribute(RuleAction):
 
     model_attribute = models.CharField(max_length=255, null=False)
     operator = models.CharField(max_length=255, choices=ChangeAttributeOperator.choices)
+    static_value = models.CharField(max_length=255, null=False)
```

Comment:
Is this field required to fill in for the user? null=False is the default value and means de static_value field is required to fill in. If this is intended behaviour then null=False could be removed.
If the static_value is not required it has to change to null=True (and likely blank=True so its value can be "")

Reply:
Good one, should be null=True and blank=True

Reply:
fixed